### PR TITLE
Tidy + bumpt to v1.0

### PR DESCRIPTION
**Pull request by gilescope**

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -189,11 +189,13 @@ fn create_ss58_registry(json: &str) -> Result<proc_macro2::TokenStream, String>
 
 		impl Ss58AddressFormat {
 			/// Network/AddressType is reserved for future use.
+			#[must_use]
```
Comment (bkchr) [inline build.rs:192]:
```suggestion
```

Comment (bkchr) [reply build.rs:192]: What is the reason for these `must_use` statements?

Please remove all of them.

File: src/lib.rs [hunk 2]
```diff
@@ -35,15 +35,19 @@ pub struct Ss58AddressFormat {
 impl Ss58AddressFormat {
 	/// Custom constructor
 	#[inline]
```
Comment (bkchr) [inline src/lib.rs:37]:
```suggestion
```

Compilers are smart enough to decide this on their own.

**Review by bkchr — APPROVED** [review]

File: src/lib.rs [hunk 3]
```diff
@@ -24,7 +24,7 @@ include!(concat!(env!("OUT_DIR"), "/account_type_enum.rs"));
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
 pub struct ParseError;
 
-/// A custom address format. See also Ss58AddressFormatRegistry
+/// A custom address format. See also `Ss58AddressFormatRegistry`
```
Comment (bkchr) [inline src/lib.rs:27]:
```suggestion
/// A custom address format. See also [`Ss58AddressFormatRegistry`]
```

File: src/lib.rs [hunk 4]
```diff
@@ -89,12 +90,12 @@ impl TryFrom<Ss58AddressFormat> for Ss58AddressFormatRegistry {
 	}
 }
 
-/// const function to convert Ss58AddressFormat to u16
+/// const function to convert `Ss58AddressFormat` to u16
```
Comment (bkchr) [inline src/lib.rs:93]:
```suggestion
/// const function to convert [`Ss58AddressFormat`] to u16
```

File: src/lib.rs [hunk 5]
```diff
@@ -89,12 +90,12 @@ impl TryFrom<Ss58AddressFormat> for Ss58AddressFormatRegistry {
...
 pub const fn from_address_format(x: Ss58AddressFormat) -> u16 {
 	x.prefix
 }
 
-/// const function to convert Ss58AddressFormat to u16
+/// const function to convert `Ss58AddressFormat` to u16
```
Comment (bkchr) [inline src/lib.rs:98]:
```suggestion
/// const function to convert [`Ss58AddressFormat`] to u16
```

File: README.md [hunk 6]
```diff
@@ -20,7 +22,19 @@ This is driven from the [json data file](src/ss58-registry.json) which contains
 
 ## Process:
 
-If you wish to add an additional account type then please raise a pull request adding to the json file.
+1. Fork and clone this repo
+
+2. Add an additional account type to `ss58-registry.json` (contiguous prefixes are better)
+
+3. Bump the patch (middle) version number of the `Cargo.toml` by running:
```
Comment (bkchr) [inline README.md:29]:
```suggestion
3. Bump the patch (last) version number of the `Cargo.toml` by running:
```

The middle is the minor

Comment (gilescope) [reply README.md:29]: ok we probably want to be bumping minor then - adding an enum arm is a minor not a patch in my head.

File: README.md [hunk 7]
```diff
@@ -20,7 +22,19 @@ This is driven from the [json data file](src/ss58-registry.json) which contains
...
+```
+cargo install cargo-bump && cargo bump patch
```
Comment (gilescope) [inline README.md:31]:
```suggestion
cargo install cargo-bump && cargo bump minor
```

File: README.md (hunk 6 above)
Comment (bkchr) [inline README.md:29]:
```suggestion
3. Bump the minor (middle) version number of the `Cargo.toml` by running:
```

**Pull request merged**